app/views/admin/users/actions/_merge.html.erb

```diff
@@ -0,0 +1,15 @@
+<div id="merge-accounts">
+  <%= form_for(@user, url: merge_admin_user_path(@user), html: { class: "flex flex-col gap-4", method: :post, onsubmit: "return confirm('Are you sure? This is extremely destructive and irreversible. Merging will delete all the other users content and combine it with this user')" }) do |f| %>
+    <p>To merge a duplicate account, make sure you are currently on the page of the user you want to KEEP. Below, add the user id of the account to merge information from, and delete.</p>
+    <div class="crayons-field mb-4">
+      <%= f.label :merge_user_id, "User ID", class: "crayons-field__label" %>
+      <p class="crayons-field__description">
+        ID of a user that will be <strong>deleted</strong> and merged.
```

Comment:
**_Quick suggestion:_** just noticed that this field description is a bit ambiguous while I was working through something on your branch! This change should hopefully make the input's description less ambiguous.
```suggestion
        ID of a user that will be <strong>deleted</strong> and merged.
```